mx.pwmc.Svgvi.json

```diff
@@ -35,8 +35,8 @@
       "sources": [
         {
           "type": "archive",
-          "url": "http://download.gnome.org/sources/vala/0.40/vala-0.40.25.tar.xz",
-          "sha256": "870fc7af58bbcfc82de6d11a96308a7bec6c180484ec431ce5bee68c2653b7df"
+          "url": "https://download.gnome.org/sources/vala/0.56/vala-0.56.2.tar.xz",
```

Comment:
The SDK has 0.56.1. Shouldn't that be enough and not have to build this?

Reply:
Yes, probably. I removed it for now. Thanks!